CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 ## Bug fixes
 
 - Fix health facilities missing from dropdown after correcting a record address [#7528](https://github.com/opencrvs/opencrvs-core/issues/7528)
+- Accessibility improvement in Choose New Password page [#5502](https://github.com/opencrvs/opencrvs-core/issues/5502)
```

Comment:
Great work on the PR! Could this be rephrased a little to communicate what the change was to other country implementors reading this? I'm sure many would appreciate this change. Maybe something like

```suggestion
- "Choose a new password" form now allows the user to submit the form using the "Enter" key [#5502](https://github.com/opencrvs/opencrvs-core/issues/5502)
```

Reply:
I have updated this, thank you @rikukissa.